src/components/course-calendar/month.cjsx

```diff
@@ -31,9 +30,22 @@ CourseMonth = React.createClass
         date: date
       )
 
+  componentDidUpdate: ->
+    @setDayHeight(@refs.courseDurations.state.groupedDurations)
+
+  setDayHeight: (groupedDurations) ->
```

Comment:
Could you just use `height: attr(data-day-height) rem;` in the CSS (not sure if that would work).

Or maybe we could decide on a max # of concurrent tasks that can be shown?